cSpell.json

```diff
@@ -244,6 +244,13 @@
                 "Droppeded"
             ]
         },
+        {
+            "filename": "**/specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/Databases.json",
+            "words": [
+                "defaut",
```

Comment:
@mohitagarwal-sql 
we need to fix it in the DSMainDev instead of adding incorrect spelling words here.

Reply:
please revert this one.

Reply:
Hi @ericshape,
The incorrect spelling words were present in auto-generated code. And it's unrelated to my changes. But I have found the incorrect spelling string in DsMainDev, and will correct the same. Thanks.

Reply:
I have reached out to owner of the code who committed the changes. Will try to get those pushed there. 

For now, I have reverted this commit.